src/core/qgsvectorlayer.h

```diff
@@ -921,6 +921,17 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
      */
     long featureCount( const QString &legendKey ) const;
 
+    /**
+     * Determines if this vector layer is empty.
+     * A layer is considered empty if either the data provider or
+     * the edit buffer have features in them.
```

Comment:
" if either the data provider or the edit buffer have features in them" -> "if neither the data provider or the edit buffer have features in them"

Reply:
neither ... nor?

Reply:
Correct! You win the good grammar award today